catalog/rest/catalog-rest-impl/src/main/java/org/codice/ddf/rest/impl/action/MetacardTransformerActionProvider.java

```diff
@@ -11,10 +11,10 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.endpoints.rest.action;
+package org.codice.ddf.rest.impl.action;
 
-import static org.codice.ddf.endpoints.rest.RESTService.CONTEXT_ROOT;
-import static org.codice.ddf.endpoints.rest.RESTService.SOURCES_PATH;
```

Comment:
❗️ We should double-check downstream projects to make sure they aren't using any of these classes directly. They'll need to be updated if they are.

To be clear, by "these classes" I mean any of the classes in here for which you're changing the package name.

Reply:
Verified by @brjeter himself. Thanks 😃